django_redis/client/default.py

```diff
@@ -813,3 +813,79 @@ def touch(
         # Convert to milliseconds
         timeout = int(timeout * 1000)
         return bool(client.pexpire(key, timeout))
+
+    def hset(
+        self,
+        name: str,
+        key: KeyT,
+        value: EncodableT,
+        version: Optional[int] = None,
+        client: Optional[Redis] = None,
+    ) -> int:
+        """
+        Set the value of hash name at key to value.
+        Returns the number of fields added to the hash.
+        """
+        if client is None:
+            client = self.get_client(write=True)
+        nkey = self.make_key(key, version=version)
+        nvalue = self.encode(value)
+        return int(client.hset(name, nkey, nvalue))
+
+    def hdel(
+        self,
+        name: str,
+        key: KeyT,
+        version: Optional[int] = None,
+        client: Optional[Redis] = None,
+    ) -> int:
+        """
+        Remove keys from hash name.
+        Returns the number of fields deleted from the hash.
+        """
+        if client is None:
+            client = self.get_client(write=True)
+        nkey = self.make_key(key, version=version)
+        return int(client.hdel(name, nkey))
+
+    def hlen(
+        self,
+        name: str,
+        client: Optional[Redis] = None,
+    ) -> int:
+        """
+        Return the number of items in hash name.
+        """
+        if client is None:
+            client = self.get_client(write=False)
+        return int(client.hlen(name))
+
+    def hkeys(
+        self,
+        name: str,
+        client: Optional[Redis] = None,
+    ) -> List:
+        """
+        Return a list of keys in hash name.
+        """
+        if client is None:
+            client = self.get_client(write=False)
+        try:
```

Comment:
why the try/catch?

Reply:
I implemented hkeys() similar to [keys()](https://github.com/jazzband/django-redis/blob/9c8cf8f32b18bf57bd684ee305e9ba6198e4ef1e/django_redis/client/default.py#L726) function. If you think that the try/catch is redundant, I can remove it.

Reply:
it's okay then, it was implemented way before I took this project, maybe we should discuss it in a separate time